wire-ios-data-model/Tests/OneOnOne/OneOnOneMigratorTests.swift

```diff
@@ -256,6 +256,117 @@ final class OneOnOneMigratorTests: XCTestCase {
         withExtendedLifetime(handler) {}
     }
 
+    func test_migrateToMLS_moveMessagesFromDuplicateProteusConversations() async throws {
+        let modelHelper = ModelHelper()
+        let sut = OneOnOneMigrator(mlsService: mockMLSService)
+        let userID = QualifiedID(uuid: .create(), domain: "local@domain.com")
+        let selfUserID = QualifiedID(uuid: .create(), domain: "local@domain.com")
+        let mlsGroupID: MLSGroupID = .random()
+
+        let selfUser = await syncContext.perform {
+            modelHelper.createSelfUser(id: selfUserID.uuid, domain: selfUserID.domain, in: self.syncContext)
+        }
+
+        let (_, proteusConversation, mlsConversation) = await createConversations(
+            userID: userID,
+            mlsGroupID: mlsGroupID,
+            in: syncContext
+        )
+
+        let duplicateProteusConversation = try await syncContext.perform {
```

Comment:
Perhaps we should have at least 2 duplicate conversations, to check that we merge messages from all proteus conversations to the mls one.